lxd/api_project.go

```diff
@@ -970,7 +970,7 @@ func projectPost(d *Daemon, r *http.Request) response.Response {
 				return fmt.Errorf("Failed loading project %q: %w", name, err)
 			}
 
-			empty, err := projectIsEmpty(ctx, project, tx)
+			empty, err := projectIsEmpty(ctx, project, tx, nil)
```

Comment:
not a blocker, but remember there are variadic arguments which means the caller doesnt have to provide the argument at all if they dont have anything to provide.

https://golangdocs.com/variadic-functions-in-golang